apps/src/p5lab/P5LabView.jsx

```diff
@@ -102,7 +110,7 @@ class P5LabView extends React.Component {
             <AnimationPicker
               channelId={this.getChannelId()}
               allowedExtensions=".png,.jpg,.jpeg"
-              libraryManifest={this.getLibraryManifest()}
+              getLibraryManifest={() => this.getLibraryManifest()}
```

Comment:
if you refactor `this.getLibraryManifest` into an arrow function, this line can simplify to:
```
getLibraryManifest={this.getLibraryManifest}
```

Reply:
(same for the case below on line 141)